github/users_gpg_keys.go

```diff
@@ -40,6 +40,28 @@ type GPGEmail struct {
 	Verified *bool   `json:"verified,omitempty"`
 }
 
+// GetGPGKeys fetches GPG keys for a user by name.
+//
+//GitHub API docs: https://developer.github.com/v3/users/gpg_keys/#list-gpg-keys-for-a-user
+func (s *UsersService) GetGPGKeys(ctx context.Context, user string) ([]*GPGKey, *Response, error) {
```

Comment:
I agree, @shurcooL - let's double-up on `ListGPGKeys` and add a `username string` parameter to it with documentation for what happens when it's empty.